src/main/java/itc/converters/ElastixSimilarity3DToAffineTransform3D.java

```diff
@@ -61,6 +61,58 @@
 	 */
 	public static AffineTransform3D convert( ElastixSimilarityTransform3D elastixSimilarityTransform3D )
 	{
-		throw new UnsupportedOperationException("PR appreciated!");
+		// fetch values from elastix transform
+		//
+		final double[][] rotationMatrix = elastixSimilarityTransform3D.getRotationMatrix();
+		final double scalingFactor = elastixSimilarityTransform3D.getScalingFactor();
+		final double[] rotationCenterInMillimeters = elastixSimilarityTransform3D.getRotationCenterInMillimeters();
+		final double[] translationInMillimeters = elastixSimilarityTransform3D.getTranslationInMillimeters();
+
+
+		// convert
+		//
+		final double[] rotationCentrePositive = new double[ 3 ];
+		final double[] rotationCentreNegative = new double[ 3 ];
+
+		for ( int d = 0; d < 3; ++d )
+		{
+			rotationCentrePositive[ d ] = rotationCenterInMillimeters[ d ];
+			rotationCentreNegative[ d ] = - rotationCenterInMillimeters[ d ];
+		}
+
+		final AffineTransform3D transform3D = new AffineTransform3D();
+
+		// rotate and scale
+		//
+
+		// translate to rotation centre
+		transform3D.translate( rotationCentreNegative );
+
+		// rotate
+		final AffineTransform3D rotate = new AffineTransform3D();
+		for ( int row = 0; row < 3; row++ )
+		{
+			for ( int col = 0; col < 3; col++ )
+			{
+				rotate.set( rotationMatrix[row][col], row, col );
```

Comment:
We need to get rid of this, cf https://github.com/image-transform-converters/image-transform-converters/issues/30, but we can take care of this in batch, so maybe it's not necessary to correct it at the moment, but for the next release, it would be great